### ISSUE TITLE
Change settings via Web

### ISSUE THREAD

**Issue by ghost** [netdata/netdata#3044]
Hello, I think it would be nice to be able to change settings via web, instead of doing the basic things via terminal. Also, have a username and password before access the monitoring dashboard would be nice.

**Comment by stefnats** [comment]
I think the second part, authentication, would (and should?) be done via apache2 / nginx reverse proxy.

**Comment by andyxh** [comment]
Thank you for your response :)

**Comment by ktsaou** [comment]
Thanks! Yes it is a nice request.

**Comment by paulfantom** [comment]
I am highly against this. Netdata is touching too many sensitive parts of the system  and modifying its configuration will cause a huge technical debt due to security concerns of such approach.

**Comment by ktsaou** [comment]
> I am highly against this. Netdata is touching too many sensitive parts of the system and modifying its configuration will cause a huge technical debt due to security concerns of such approach.

netdata settings, not system settings. For example, it would be very nice to be able to:

1. create or modify alarms
2. setup alarm notifications 
3. restart auto-detection of a data-collection plugin
4. configure data collection plugins

Of course authentication is required.

**Comment by ktsaou** [comment]
@cakrit 	this is probably a feature-set for the hub.

**Comment by cakrit** [comment]
Blocked by #6317.

**Comment by cakrit** [comment]
Duplicate of #6432
